### ISSUE TITLE
Add fetch command to Repository explorer

### ISSUE THREAD

**Issue by escamoteur** [eamodio/vscode-gitlens#470]
Hi,

just installed Gitlens and it looks great!

My working folder is currently behind the remote repository what I can see when looking at github, but that isn't signaled in gitlens. Also if I call compare with working folder in the Remotes branch nothing is displayed.
Any help would be awesome.
Thanks
Thomas

**Comment by escamoteur** [comment]
Thanks a lot, that fixed it. Not sure if this is mentioned somewhere in your docs.

**Comment by eamodio** [comment]
Glad that resolved it. You are right, it isn't mentioned in the docs -- I probably should add that, or maybe I should execute a fetch on refresh, if `git.autofetch` isn't enabled.

**Comment by escamoteur** [comment]
Yes would be good just got the feedback from someone else with the same problem. I fear a lot of users might install gitlense and directly uninstall it again because they think it doesn't work.

**Comment by GargantulaKon** [comment]
Yes, I just had the same issue. I'm more used to a GUI for Git and by clicking the refresh icon in the GitLens sidebar, I assumed that it would do a fetch for me. I checked the Git output logs and I don't see it doing so.

![image](https://user-images.githubusercontent.com/3610651/43841101-5bcad2dc-9af0-11e8-8be5-5c4a56a5cedc.png)

In addition, another related issue was that I was trying to switch to a new branch and I wanted to see it in the GitLens sidebar. I clicked on the refresh icon and I expected the branch to appear under the Remote section. It didn't see it so I ran a git fetch on the command line and then it appeared after I clicked on the refresh icon.

I'm not sure if you want to add a separate Git fetch button or make it part of the refresh button, but I could also see users getting confused by that.
